src/content/api/cli.mdx

```diff
@@ -608,3 +608,18 @@ To use `webpack v5.32.0`:
 ```bash
 WEBPACK_PACKAGE=webpack-5 npx webpack
 ```
+
+## Troubleshooting
+
+### TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for ./webpack.config.ts
+
+You might encounter this error in the case of typescript ECMA modules (i.e. `type: "module"` in `package.json`).
```

Comment:
```suggestion
You might encounter this error in the case of using native ESM in TypeScript (i.e. `type: "module"` in `package.json`).
```
Either use `ECMAScript modules` or `ESM`, I don't think `ECMA modules` is right.